roles/common/defaults/main.yml

```diff
@@ -445,6 +445,22 @@ gi_patches:
   - { category: "RU", base: "21.3.0.0.0", release: "21.16.0.0.0", patchnum: "36990664", patchfile: "p36990664_210000_Linux-x86-64.zip", patch_subdir: "/", prereq_check: false, method: "opatchauto apply", ocm: false, upgrade: false, md5sum: "N1p/HLEg+UFYAr04loujqg==" }
   - { category: "RU", base: "21.3.0.0.0", release: "21.17.0.0.0", patchnum: "37349593", patchfile: "p37349593_210000_Linux-x86-64.zip", patch_subdir: "/", prereq_check: false, method: "opatchauto apply", ocm: false, upgrade: false, md5sum: "Mt0Bw+IPKqoKh31YkneCrg==" }
 
+# 21c GRID RU - 21.4 and 21.7 are available only via SR
```

Comment:
Here too;  do we intend to modify patch definitions for 21.3 here?

Reply:
Not sure why that was changed here, likely a mistake.  Reverted.